/hold

@BrendanSChang 

```
Errors from golint:
test/fuzz/yaml/yaml.go:30:1: comment on exported function FuzzDurationStrict should be of the form "FuzzDurationStrict ..."
test/fuzz/yaml/yaml.go:50:1: comment on exported function FuzzMicroTimeStrict should be of the form "FuzzMicroTimeStrict ..."

Please review the above warnings. You can test via "golint" and commit the result.
If the above warnings do not make sense, you can exempt this package from golint
checking by adding it to hack/.golint_failures (if your reviewer is okay with it).
```

--- a/test/fuzz/yaml/yaml.go
+++ b/test/fuzz/yaml/yaml.go
@@ -27,8 +27,8 @@ import (
 	sigyaml "sigs.k8s.io/yaml"
 )
 
-// FuzzDuration is a fuzz target for strict-unmarshaling Duration defined in
-// "k8s.io/apimachinery/pkg/apis/meta/v1". This target also checks that the
+// FuzzDurationStrict is a fuzz target for strict-unmarshaling Duration defined
+// in "k8s.io/apimachinery/pkg/apis/meta/v1". This target also checks that the
 // unmarshaled result can be marshaled back to the input.
 func FuzzDurationStrict(b []byte) int {
 	var durationHolder struct {
@@ -47,9 +47,9 @@ func FuzzDurationStrict(b []byte) int {
 	return 1
 }
 
-// FuzzMicroTime is a fuzz target for strict-unmarshaling MicroTime defined in
-// "k8s.io/apimachinery/pkg/apis/meta/v1". This target also checks that the
-// unmarshaled result can be marshaled back to the input.
+// FuzzMicroTimeStrict is a fuzz target for strict-unmarshaling MicroTime
+// defined in "k8s.io/apimachinery/pkg/apis/meta/v1". This target also checks
+// that the unmarshaled result can be marshaled back to the input.
 func FuzzMicroTimeStrict(b []byte) int {
 	var microTimeHolder struct {
 		T metav1.MicroTime `json:"t"`

